theano/__init__.py

```diff
@@ -31,10 +31,6 @@
 
 import sys
 
-if sys.platform == 'win32' and sys.version_info[0:2] == (3, 5):
-    raise RuntimeError(
-        "Theano do not support Python 3.5 on Windows. Use Python 2.7 or 3.4.")
```

Comment:
I think the Python build with MSVC (so the official python 3.5 on Windows) is still not supported. Instead of just removing it, can you skip this error when MSVC wasn't used? I don't know how to detect that, if you also don't know, you could skip that error for anaconda only.